### PR TITLE
Release pipeline

### PR DESCRIPTION
**Pull request by wkozyra95**
Release process for Linux binaries + docs

### CODE REVIEW

**Comment by wkozyra95** [comment]
Merging because I need to verify some things on master, I'll address any comments in follow up.

File: examples/web_renderer.rs [hunk 1]
```diff
@@ -38,8 +38,11 @@ fn main() {
         .unwrap()
         .join("..");
 
-    if cef::bundle_app(target_path).is_err() {
-        panic!("Build process helper first: cargo build --bin process_helper");
+    if let Err(err) = bundle_for_development(target_path) {
+        panic!(
+            "Build process helper first: cargo build --bin process_helper. {:?}",
+            err
```
Comment (WojciechBarczynski) [inline examples/web_renderer.rs:44]: Maybe add note about `-r`

File: src/bin/package_for_release/bundle_macos.rs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            "-czvf",
+            output_name,
+            "video_compositor.app",
+        ])
+        .spawn()?
+        .wait()?
+        .code();
+    if exit_code != Some(0) {
+        return Err(anyhow!("Command tar failed with exit code {:?}", exit_code));
+    }
```
Comment (WojciechBarczynski) [inline src/bin/package_for_release/bundle_macos.rs:76]: The exit code itself isn't really helpful. Can we collect the result and add it here? (like the first element returned from: https://hexdocs.pm/elixir/System.html#cmd/3)

**Review by WojciechBarczynski — COMMENTED** [review]
LGTM

File: src/bin/package_for_release/bundle_macos.rs (hunk 2 above)
Comment (wkozyra95) [reply src/bin/package_for_release/bundle_macos.rs:76]: stdout and stderr is already printed to the terminal.

Comment (WojciechBarczynski) [reply src/bin/package_for_release/bundle_macos.rs:76]: Ok

**Pull request merged**